src/Marten/Linq/Includes/TemporaryTableStatement.cs

```diff
@@ -30,6 +29,7 @@ protected override void configure(ICommandBuilder sql)
     {
         sql.Append("drop table if exists ");
         sql.Append(ExportName);
+        sql.Append("; ");
```

Comment:
You no longer need the semicolon between commands with the newer NpgsqlBatch usage.